### DISCUSSION TITLE
How to get the nice TUI panels as per screenshots?

### DISCUSSION THREAD

**Discussion by NickLarsenNZ** [tokio-rs/console#456] [General]
When I run `tokio-console` (`0.1.9`), I seem to only get a plain text render with no panels.

<img width="1099" alt="image" src="https://github.com/tokio-rs/console/assets/10092581/d0e12ba7-6e93-4f9b-a059-e412e6eacbc4">

I have tried `--ascii-only` (`true` and `false`) and `--colorterm`.

My terminal is Alactritty on macOS.

**Comment by hds** [comment]
Tokio console bases terminal support for UTF-8 on the `LANG` environment variable (which is pretty common). If it finds a value that ends in `UTF-8` then it will display the pretty UI that requires UTF-8 characters. This value can also be specified with the `—lang` command line flag.

So try something like `tokio-console —lang=de_DE.UTF-8`.
